packages/governance/test/swingsetTests/contractGovernor/governParam.js

```diff
@@ -0,0 +1,72 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { q, details as X } from '@agoric/assert';
+
+import { ChoiceMethod } from '../../../src/ballotBuilder';
+
+const setupGovernance = (paramManager, registrar) => {
+  const params = E(paramManager).getParams();
+  const validNames = Object.keys(params);
+
+  /**
+   * @param {string} name
+   * @param {ParamValue} proposedValue
+   * @param {ClosingRule} closingRule
+   */
+  const setParamByVoting = async (name, proposedValue, closingRule) => {
+    assert(
+      validNames.includes(name),
+      X`Name ${name} was not a governable parameter`,
+    );
+    /** @type {ParamDescription["type"]} */
+    // TODO: use paramDescription.type to actually enforce the type
+    // const paramType = params[name].type;
+
+    const positivePosition = {
+      text: `Change ${name} to ${q(proposedValue)}.`,
+      proposedValue,
+    };
+    const negativePosition = {
+      text: `Leave ${name} unchanged.`,
+    };
+
+    const binaryBallotDetails = {
+      type: ElectionType.BINARY,
+      method: ChoiceMethod.CHOOSE_N,
+      questions: `Change ${name} in ${q(governedInstance)}?`,
+      positions: [positivePosition, negativePosition],
+      maxChoices: 1,
+      quorumThreshold: QuorumRule.HALF,
+      tieOutcome: negativePosition,
+      closingRule,
+    };
+
+    const outcomeP = E(registrar).askQuestion(binaryBallotDetails);
```

Comment:
This is fanciful, but I was wondering if we could present a simpler API, such that there's just one object (here, called `registrar`) that we call `askQuestion` on with all of the ballot details. We have a number of different pieces, and it seems like most of them are being exposed to the user, but can some pieces encapsulate others so that the external interface is simpler?